internal/cmd/db.go

```diff
@@ -109,6 +109,14 @@ var dbCmd = &cobra.Command{
 
 const ENV_ACCESS_TOKEN = "TURSO_API_TOKEN"
 
+type notLoggedInError struct{}
+
+func (*notLoggedInError) Error() string {
+	return fmt.Sprintf("user not logged in, please login with %s", internal.Emph("turso auth login"))
+}
+
+var NotLoggedInError = &notLoggedInError{}
```

Comment:
wouldn't following just suffice:

```suggestion
var NotLoggedInError = fmt.Errorf("user not logged in, please login with %s", internal.Emph("turso auth login"))
```

Reply:
I created a new type on purpose. See how type is not exported/public. This way I make sure no one creates by accident an error that matches NotLoggedInError.

Reply:
I'm committing the PR now but let's discuss further if necessary. We can change that in follow-up if we decide to.